samples/openapi3/client/petstore/go/go-petstore/docs/Order.md

```diff
@@ -7,7 +7,7 @@ Name | Type | Description | Notes
 **Id** | Pointer to **int64** |  | [optional] 
 **PetId** | Pointer to **int64** |  | [optional] 
 **Quantity** | Pointer to **int32** |  | [optional] 
-**ShipDate** | Pointer to [**time.Time**](time.Time.md) |  | [optional] 
+**ShipDate** | Pointer to [**time.Time** |  | [optional] 
```

Comment:
@therve thanks for the PR. Looks like it's missing a closed square bracket.

Reply:
Good catch, thanks! It was the other way around, there was an additional opening bracket, should be fixed.